server/data/data-cassandra/src/test/java/org/apache/james/rrt/cassandra/RewriteTablesTest.java

```diff
@@ -31,6 +29,5 @@
         glue = { "org.apache.james.rrt.lib", "org.apache.james.rrt.cassandra" }
     )
 public class RewriteTablesTest {
-    
-    @ClassRule public static DockerCassandraRule cassandraServer = new DockerCassandraRule();
+
```

Comment:
is it normal to remove everything here?

Reply:
Yes it !

The `CassandraStepdefs` can directly use the singleton so that test don't need the classrule anymore.